templates/neutronapi/httpd/10-neutron-httpd.conf

```diff
@@ -1,4 +1,8 @@
+{{ range $endpt, $vhost := .VHosts }}
+# {{ $endpt }} vhost {{ $vhost.ServerName }} configuration
 <VirtualHost *:9696>
```

Comment:
shouldn't we use port 13696 if TLS is enabled? IIRC this is what we are using in e.g. OSP 17.1

Reply:
why you think we should do this? its just an internal port on the httpd behind a k8s service.

Reply:
ok, so in such case we don't need to do it like that. You are right